projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -1947,7 +1951,7 @@ public function record_tracks_event( $event_name, $event_props ) {
 				}
 			}
 
-			$tracking = new \Automattic\Jetpack\Tracking();
+			$tracking = new Tracking();
```

Comment:
I updated this since we're now importing the the class at the top of the file.

Just above this existing tracking code there's a conditional check for if the user is connected on self-hosted sites. I wonder if should go ahead and replace that check with the method created below, ie `if { self::can_use_analytics ) {}`? That new method checks user connection status among other things. 

https://github.com/Automattic/jetpack/blob/1614a012a3ffcaf66b28b7d1ccb00f4f68e8bc0c/projects/packages/forms/src/contact-form/class-contact-form-plugin.php#L1943